ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/CustomDictionaryStyle/CustomDictionaryStyle.qml

```diff
@@ -31,4 +31,37 @@ Item {
         id: model
         mapView: view
     }
+
+    Rectangle {
+        id: rectangle
+        x: 5; y: 5
```

Comment:
try to avoid using x/y directly and instead use anchors

```
Rectangle {
    id: rectangle
    anchors {
        left: parent.left
        top: parent.top
        margins: 5
    ...
```